Src/AutoFoq/AutoFoqCustomization.fs

```diff
@@ -18,7 +18,8 @@ type AutoFoqCustomization(relay: ISpecimenBuilder) =
         AutoFoqCustomization(
             FilteringSpecimenBuilder(
                 MethodInvoker(
-                    FoqMethodQuery()),
+                    FoqMethodQuery(
+                        Fixture())),

```

Comment:
To be honest, I was going a bit back and forth on that one :neutral_face:

In the beginning I tried to move the whole object graph in the `Customize` method, in order to supply the given `IFixture` instance, but then I got failing tests which were not easy to make them passing.

---

Then I thought that supplying a *default* `Fixture` can be considered an implementation detail; just so that Foq has something reasonable to return. – After all, this affects *only* those Test Doubles that have not being explicitly setup.

So I went by supplying a default `Fixture` instance, but I've also added a façade test [here](https://github.com/moodmosaic/AutoFixture/blob/use-autofoq-return-strategy-to-avoid-nulls/Src/AutoFoqUnitTest/FixtureIntegrationTest.fs#L82-L95) to demonstrate that it's possible to use the types defined in AutoFoq with a customized `Fixture` instance.